frontend/packages/wizard/src/wizard.tsx

```diff
@@ -135,11 +137,32 @@ const Wizard = ({
   };
 
   const handleNext = () => {
-    dispatch(WizardAction.NEXT);
+    dispatch({ type: WizardActionType.NEXT });
+  };
+
+  const handleStepClick = (step: number) => {
+    dispatch({ type: WizardActionType.GO_TO_STEP, step });
+  };
+
+  const handleNavigation = (
+    params: { toOrigin?: boolean; keepSearch?: boolean },
```

Comment:
Once we create the interface in the wizard-context, we can export it for use here so we don't have to keep up with the different places its typed.